easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -588,6 +592,29 @@ def sanity_check_step(self, *args, **kwargs):
         else:
             self.log.debug("Detection of downloaded dependencies not enabled")
 
+        # inject directory path that uses %(pyshortver)s template into default value for sanity_check_paths,
+        # but only for stand-alone installations, not for extensions;
+        # this is relevant for installations of Python packages for multiple Python versions (via multi_deps)
+        # (we can not pass this via custom_paths, since then the %(pyshortver)s template value will not be resolved)
+        if not self.is_extension and not self.cfg['sanity_check_paths']:
+            self.cfg['sanity_check_paths'] = {
+                'files': [],
+                'dirs': [os.path.join('lib', 'python%(pyshortver)s', 'site-packages')],
```

Comment:
Doesn't this mean that the sanity check is just performed in a single directory (eg: `python2.7`), instead of all the directories needed when using `multi_deps`?

Reply:
No, because in https://github.com/easybuilders/easybuild-framework/pull/2813 the `sanity_check_step` is done iteratively, once for every entry in `multi_deps`.

When `Python` 2.7.x is listed as a build dep, `%(pyshortver)s` will resolve to `2.7`; when `Python` 3.6.x is used, it will resolve to `3.6`.